config/dm_config/nnf-dm-config.yaml

```diff
@@ -23,6 +23,16 @@ profiles:
       # default: command: mpirun --allow-run-as-root --hostfile $HOSTFILE dcp --progress 1 --uid $UID --gid $GID $SRC $DEST
       command: mpirun --allow-run-as-root --hostfile $HOSTFILE dcp --progress 1 --uid $UID --gid $GID $SRC $DEST
 
+      # If true, enable the command's stdout to be saved in the log when the command completes
+      # successfully. On failure, the output is always logged.
+      # Note: Enabling this option may degrade performance.
```

Comment:
The "On failure, the output is always logged.", makes me wonder what this "Enabling this option may degrade performance." means, and exactly when the performance is affected.

I think you should delete that "Note...".

Reply:
Removed.